framework/logging.cpp

```diff
@@ -1926,6 +1933,8 @@ void TapFormatLogger::print(int tc)
         extra += format_status_code();
         break;
     case TestResult::OutOfMemory:
+        extra += "Out of memory";
+        break;
```

Comment:
While correct, we didn't use to do this and should probably leave it as it used to for TAP.